content/blog/create-a-cold-tiered-volume-and-use-hpe-ezmeral-data-fabric-object-store-as-remote-target.md

```diff
@@ -0,0 +1,656 @@
+  AccessKey :
+  SecretKey :
+  API       :
+  Path      : auto
+
+play
+  URL       : https://play.min.io
+  AccessKey : XXXXXXXXXXXXXXXXXXXX
+  SecretKey : XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+  API       : S3v4
+  Path      : auto
+
+s3
+  URL       : https://s3.amazonaws.com
+  AccessKey : YOUR-ACCESS-KEY-HERE
+  SecretKey : YOUR-SECRET-KEY-HERE
+  API       : S3v4
+  Path      : dns
+```
+
+##### 2. Generate S3 keys to authenticate your administrator
+
+The cluster administrator (typically the mapr user) must authenticate to the Object Store cluster and generate S3 keys (accessKey and secretKey) on the default Object Store account.
+Perform this operation before performing any CLI operations in Object Store.
+
+If the cluster is secure, use [maprlogin](https://docs.datafabric.hpe.com/72/SecurityGuide/ThemaprloginUtility.html) to authenticate the cluster administrator, and then generate the keys:
+
+```shell
+maprcli s3keys generate -domainname primary -accountname default -username mapr -json
+```
+
+🗒Note: An Object Store cluster has a domain, accounts, buckets, users, and access policies associated with it.
+Installing Object Store in a cluster provides a primary domain and a default account.
+
+Sample output:
+
+```json
+{
+  "timestamp":1676472096994,
+  "timeofday":"2023-02-15 10:41:36.994 GMT+0800 PM",
+  "status":"OK",
+  "total":1,
+  "data":[
+          {
+            "accesskey":"XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX",
+            "secretkey":"XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
+          }
+  ]
+}
+```
+
+🗒Note: If you encounter any problem when generating the S3 keys, refer to this page: [Generate S3 Keys to Authenticate Users and Applications](https://docs.datafabric.hpe.com/72/MapROverview/object-store-get-started.html#object-store-get-started__section_x4h_w13_4tb).
+
+##### 3. Use the `mc alias set` command to create an alias for admin user
+
+```shell
+mc alias set s3-admin-alias https://`hostname -f`:9000 {ACCESS_KEY} {SECRET_KEY} --api "s3v4" --path "off" --json
+```
+
+🗒Note: "s3-admin-alias" is the name of the alias, you define it.
+
+"https://`hostname -f`:9000" is the endpoint of the Object Store service.
+Here, I'm running the command on the node which is running the S3server.
+After created an alias, you would find the information is appended into <ins>$HOME/.mc/config.json</ins>.
+
+##### 4. Create an account
+
+I choose to use the Object Store Web GUI to create an account.
+
+Refer to this document - [Using the Object Store Interface](https://docs.datafabric.hpe.com/72/MapROverview/create-account.html#create_account__section_fsm_1hn_nrb).
+
+Create an account using Object Store Web GUI - 1
+<a href="https://ibb.co/SyW9MKK"><img src="https://i.ibb.co/XLNvKzz/Object-Store-Create-Account-1.png" alt="Object-Store-Create-Account-1" border="0"></a>
+
+Enter the following in "Default Bucket Place":
```

Comment:
should this be "default bucket place" or is "Default Bucket Place" showing on the screen?